lib/blather/stanza/x.rb

```diff
@@ -254,22 +254,25 @@ def label=(label)
       #
       # @param [String]
       def value
-        if self.namespace
-          content_from 'ns:value', :ns => self.namespace.href
+        children = if self.namespace
+          xpath('ns:value', ns: self.namespace.href)
         else
-          content_from :value
+          xpath(:value)
         end
+        return children.first&.content if children.length < 2
+        children.map(&:content)
       end
 
       # Set the field's value
       #
       # @param [String] value the field's value
       def value=(value)
         self.remove_children :value
-        if value
+        return unless value
+        Array(value).each do |val|
```

Comment:
It looks like this allows passing in an array of values rather than a single value, and casting each array element to an XMPPNode. Would you update the method docs to reflect this new behavior, and add a supporting spec?

Reply:
Added spec, changed comment